test/python/WMCore_t/WorkQueue_t/WorkQueue_t.py

```diff
@@ -602,44 +611,43 @@ def testQueueChaining(self):
                                        {})
         self.assertEqual(0, len(self.localQueue))
         self.assertEqual(2, len(work))
-
+ 
```

Comment:
When return key is hit it automatically indent the line with space. I will clean that up